packages/web-components/@sfx/products/src/products.ts

```diff
@@ -69,9 +69,17 @@ export default class Products extends LitElement {
           flex-wrap: wrap;
         }
 
+        sfx-products[hidden] {
+          display: none;
+        }
+
         sfx-product {
```

Comment:
Oh, this is where this style ended up. This should live in the `Product` component. That said, if we want to push that off to the future base styles ticket, we can do so as well.

Reply:
We'll deal with this in a future ticket.